internal/errors/README.md

```
@@ -0,0 +1,445 @@
+        Category(errors.CategoryDatabase).
+        Timing("save_detection", time.Since(start)).
+        Build()
+}
+```
+
+## Sentry Integration
+
+### Error Titles
+
+The system generates meaningful Sentry titles based on:
+1. Component name (e.g., "Imageprovider")
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)